modules/ROOT/pages/deployment/general/roles-permissions.adoc

```diff
@@ -0,0 +1,128 @@
+
+Consider that these roles and permissions are provided by the system via APIs, which means you can call them via your own code or `curl`. 
+
+Read the image below the following way::
+* A Role gets one or more permission types assigned.
+* Each permission type grants individual rights relevant for that role.
+* Roles are assigned to users.
+
+// the empty macro is necessary to have more distance between text and image
+ 
+{empty} +
+
+// the table references following ocis code where roles and permissions a re defined: https://github.com/owncloud/ocis/blob/master/services/settings/pkg/store/defaults/defaults.go
+
+image::deployment/general/roles-and-permissions-matrix.svg[width=550]
+
+=== System Role Details
+
+In general with regards to spaces, the following applies:
+
+* Spaces can always be deleted by space managers - with the exception of the `Delete Home Space` permission, which is only assigned to the Admin since this should only happen when a user account gets deleted.
+* Managing a space means creating or deleting a space, setting quotas, delegating the right to manage space members and space member roles, but not having access to the content of the space except when being a member of the space.
+* When creating a space, the creator is automatically in the delegatable Manager role for this space. For details see xref:space-roles-and-permissions[Space Roles and Permissions].
+
+Admin::
+The `Admin` role provides full control of all permission types with one exception, which is `Self Management`. This permission type is excluded to avoid that the last remaining admin accidentally reduces their own role to a lower one making the system unmanageable. As long as there is more than one admin, roles can be set via the `Role Management` permission.
+
+Space Admin::
+The `Space Admin` role allows you to create spaces, set quotas and delegating the right to manage space members and space member roles.
+
+User::
+If you have the `User` role assigned, you can manage your own account and change the language.
+
+Guest::
+Users with the `Guest` role assigned can only change their language.
+
+== Space Roles and Permissions
+
+// https://github.com/owncloud/web/blob/3ad8561a1c7ac5ea4f1b30397569f4b9c18166f7/packages/web-pkg/src/services/permissionManager.ts
+
+When it comes to permissions for a space, three roles are available:
+
+* Manager
+* Editor
+* Viewer
+
+Note that the creator of a space becomes manager automatically, can add other users and delegate his Manager role to one or more space members.
+
+The following table describes the space roles and their permissions. Note that permissions are added the higher the role gets.
+
+[width=60,cols=".^30%,100%",options="header",]
+|===
+| Role
+| Permissions
+
+| Viewer
+a| * read documents,
+* view images,
+* play videos,
+* get info about the space.
+
+| Editor
+a| * modify documents and even delete them,
+* upload content to a space,
```

Comment:
do we want to mention move/rename?
the ability to download, delete and upload kind of implies that the holder has the ability to rename or move a resource, because rename or move can always be done the inefficient way anyhow, b y downloading the resource, deleting it, then upload to the new location/name.